src/installation/guides/index.md

```diff
@@ -3,4 +3,7 @@
 Got weird hardware or an unusual setup? This is the section for guides about how
 to install Void on things that it doesn't usually like.
```

Comment:
Use this opportunity to change this to something else? Like "to install Void on things that are not as straightforward" (I'm not married to this suggestion, far from it). Just don't like that it implies that Void isn't adequate for being used in these situations.

@flexibeast any thoughts?

Reply:
@ericonr: i agree. How about: "This section contains guides for more specific or complex use-cases." But i'm not married to this phrasing either. :-)

Reply:
I do like yours!

Reply:
sounds good